packs/actions/erupting-spurs.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "offensive",
         "description": {
-            "value": "<p><strong>Trigger</strong> A creature takes persistent bleed damage from your bone flense spell</p><hr /><p><strong>Effect</strong> You will the bone spurs to erupt in a particularly devastating manner, dealing @Damage[6d6[piercing]] damage (basic Fortitude save) to the triggering creature instead of the persistent bleed damage from bone flense. The duration of bone flense immediately ends.</p>"
+            "value": "<p><strong>Trigger</strong> A creature takes persistent bleed damage from your bone flense spell</p><hr /><p><strong>Effect</strong> You will the bone spurs to erupt in a particularly devastating manner, dealing @Damage[((6 + 4*((@actor.flags.pf2e.boneFlense.rank - 3)/2)))d6[piercing]] damage (basic Fortitude save) to the triggering creature instead of the persistent bleed damage from bone flense. The duration of bone flense immediately ends.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Trigger</strong> A creature takes persistent bleed damage from your bone flense spell</p><hr /><p><strong>Effect</strong> You will the bone spurs to erupt in a particularly devastating manner, dealing @Damage[6d6[piercing]] damage (basic Fortitude save) to the triggering creature instead of the persistent bleed damage from bone flense. The duration of bone flense immediately ends.</p>"
```